### PR TITLE
Jb50790

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: buildqtc.sh [hunk 1]
```diff
@@ -428,6 +428,17 @@ build_windows_qtc() {
...
+        VCVARSALL="%_programs%\microsoft visual studio $DEF_MSVC_VER_ALT\vc\vcvarsall.bat"
+        D3DCOMPILER="%_programs%\microsoft visual studio $DEF_MSVC_VER_ALT\vc\bin\D3Dcompiler_*.dll"
+        VCREDISTCRT="%_programs%\microsoft visual studio $DEF_MSVC_VER_ALT\vc\redist\x86\microsoft.vc*.crt"
+    else
+        VCVARSALL="%_programs%\Microsoft Visual Studio\\$DEF_MSVC_VER\BuildTools\VC\Auxiliary\Build\vcvarsall.bat"
+        VCVARSALL_PARAMS="x86"
+        D3DCOMPILER="%VCToolsInstallDir%\bin\Hostx86\x86\d3dcompiler_*.dll"
+        VCREDISTCRT="%VCToolsRedistDir%\x86\Microsoft.VC*.CRT"
+    fi
+
```
Comment (martyone) [inline buildqtc.sh:441]: I would prefer simply updating the literal values for the version we currently use. Should someone need to use other version, they should do the required modifications in their own Git branch.

**Comment by vigejolla** [comment]
> Now that you are about to rebuild/upgrade Qt, it would be nice not to merge this right now and possibly exclude the second commit if it is not needed then. (I am sure you know :))

That's what I thought as well :)

File: buildqt5.sh [hunk 2]
```diff
@@ -119,6 +119,7 @@ if Not DEFINED ProgramFiles(x86) set _programs=%ProgramFiles%
 
 set PATH=c:\windows;c:\windows\system32;%_programs\windows kits\8.0\windows performance toolkit;%_programs%\7-zip;$(win_path $DEF_PREFIX)\invariant\bin;c:\python27;c:\perl\bin;c:\ruby193\bin;$(win_path $DEF_ICU_INSTALL_DIR)\bin;$(win_path $DEF_QT_SRC_DIR)\gnuwin32\bin;%_programs%\microsoft sdks\typescript\1.0;c:\windows\system32\wbem;c:\windows\system32\windowspowershell\v1.0;$(win_path $DEF_PREFIX)\invariant\bin
 call "%_programs%\microsoft visual studio $DEF_MSVC_VER_ALT\vc\vcvarsall.bat" || exit 1
+call "%_programs%\Microsoft Visual Studio\\$DEF_MSVC_VER\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x86 || exit 1
```
Comment (martyone) [inline buildqt5.sh:122]: Hm, shouldn't this be a replacement instead of addition?

Comment (vigejolla) [reply buildqt5.sh:122]: It should indeed

**Pull request merged**